android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIText.java

```diff
@@ -214,8 +214,34 @@ public void processProperties(KrollDict d)
 		if (d.containsKey(TiC.PROPERTY_AUTO_LINK)) {
 			TiUIHelper.linkifyIfEnabled(tv, d.get(TiC.PROPERTY_AUTO_LINK));
 		}
+		
+		if (d.containsKey(TiC.PROPERTY_PADDING)) {
+			setTextPadding((HashMap)d.get(TiC.PROPERTY_PADDING));
+		}
+		
 	}
 
+	private void setTextPadding(HashMap<String, Object> d)
+	{
+		int paddingLeft = 0;
+		int paddingRight = 0;
+		
+		if (d.containsKey(TiC.PROPERTY_LEFT)) {
+			paddingLeft = TiConvert.toInt(d, TiC.PROPERTY_LEFT);
```

Comment:
Please use TiConvert methods that with a fall back default in case the input is invalid like null and such. Something like this TiConvert.toInt(d, TiC.PROPERTY_LEFT, someDefaultValue).